src/nvim/lua/treesitter.c

```diff
@@ -725,6 +725,7 @@ static struct luaL_Reg node_meta[] = {
   { "descendant_for_range", node_descendant_for_range },
   { "named_descendant_for_range", node_named_descendant_for_range },
   { "parent", node_parent },
+  { "child_containing_descendant", node_child_containing_descendant },
```

Comment:
is `TSNode` intended to directly mirror the TS api? or can we choose better names?